packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -58,28 +58,62 @@ type Props = {
  * The "standard" modal layout: a titlebar, a content area, and a footer.
  */
 export default class StandardModal extends React.Component<Props> {
+    /** A unique HTML ID for the title element. */
+    _titleId: string;
+
+    /** A unique HTML ID for the subtitle element. */
+    _subtitleId: string;
+
     static defaultProps = {
         onClickCloseButton: () => {},
     };
 
+    constructor(props: Props) {
+        super(props);
+
+        // Generate a unique ID for this modal. Math.random() isn't great, but
+        // collisions on the same page should be pretty unlikely.
+        //
+        // TODO(mdr): Once we add snapshot testing, will this break it? We might
+        //     need to figure out how to seed the PRNG, or mock Math.random, or
+        //     find a different approach altogether.
+        const modalId = Math.random()
+            .toString(36)
+            .substr(2);
```

Comment:
Maybe we can extract this into a separate module and then do something similar to what we do for aphrodite styles with the `SNAPSHOT_INLINE_APHRODITE` environment variable in `package.json`.

Reply:
The reason for suggesting a separate module, `wonder-blocks-utils` for example, is that we'll probably need to do the same thing in other modules/components as well.

Reply:
I guess at the very least, put it in a separate file with a TODO to make it a separate component later.